email/1.3/modules/ROOT/edits/email-trigger.adoc

```diff
@@ -0,0 +1,219 @@
+= Trigger a Flow When Receiving a New Email Examples - Mule 4
+:page-aliases: connectors::email/email-trigger.adoc
+
+The following examples illustrate how you configure the Anypoint Connector for Email (Email Connector) source listeners *On New Email - IMAP* and *On New Email - POP3* to identify and retrieve unread email messages, initiating a flow.
+
+== Configure Delete After Retrieve Parameter
+
+Email Connector *On New Email - IMAP* and *On New Email - POP3* sources support the *Delete after retrieve* parameter that enables the deletion of retrieved emails from the mailbox folder after the operation process is complete. By deleting processed emails, you can ensure that the next processed emails are new ones. Because the POP3 protocol doesn't support watermarking, the *Delete after retrieve* parameter is commonly used with this protocol. +
```

Comment:
The second sentence might be a bit hard to quickly parse, keeping straight "the processed emails" vs. "the next processed emails." What do you think about something like this...:
```suggestion
Email Connector *On New Email - IMAP* and *On New Email - POP3* sources support the *Delete after retrieve* parameter that enables the deletion of emails from the mailbox after the operation that they set in motion is complete, ensuring that the next operation is based on newly received emails. Because the POP3 protocol doesn't support watermarking, the *Delete after retrieve* parameter is commonly used with this protocol. +
```
It might be better to lead up to the parameter in this way:
1. POP3 doesn't support watermarking.
2. This makes identifying emails belonging to separate operations difficult if they congregate in the mailbox.
3. To ensure that the emails in the mailbox are current, enable *Delete after retrieve*, so that only unprocessed email resides in the mailbox as a flow proceeds.

problem/reason > solution

Reply:
QUESTION: The parameter is named `Delete after retrieve` but it sounds like the email is NOT deleted after retrieval so much as after the process that it initiates is completed. Do we need to clarify this?